cypress/e2e/tests/pages/explorer/workloads/jobs.spec.ts

```diff
@@ -29,10 +34,44 @@ describe('Cluster Explorer', { tags: ['@explorer', '@adminUser'] }, () => {
 
         workloadsJobsListPage.listElementWithName(jobName).should('exist');
 
-        // navigate to namespace and check existance of namespace
+        // navigate to namespace and check existence of namespace
         cy.visit('/c/local/explorer/projectsnamespaces');
         workloadsJobsListPage.listElementWithName(namespaceName).should('exist');
       });
+
+      it('Should be able to clone a job', () => {
+        const jobName2 = `${ jobName }-2`;
+        const jobNameClone = `${ jobName }-3`;
```

Comment:
Would these names possibly collision on a Cypress retry? Otherwise this overall looks good to me.